rclpy/threading/data_plotter.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from copy import deepcopy
+from threading import Lock, Thread
+import time
+
+import rclpy
+
+import matplotlib.pyplot as plt
```

Comment:
Just use the roadworks entry for matplotlib for now: https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L1061